content/blog/newsletter/2022/01/2023-02-05-newsletter.md

```diff
@@ -0,0 +1,220 @@
+---
+title: The XMPP Newsletter December 2022 & January 2023
+date: 2023-02-05
+categories: ['Newsletter']
+---
+
+Welcome to the XMPP Newsletter, great to have you here again! This issue covers the month of December 2022 & January 2023. 
+This is the first release of this year and we have after a well-deserved winter break! Many thanks to all our readers and all contributors!
+
+Like this newsletter, many projects and their efforts in the XMPP community are a result of people’s voluntary work. If you are happy with the services and software you may be using, especially throughout the current situation, please consider saying thanks or help these projects! Interested in supporting the Newsletter team? Read more at the bottom.
+
+### Newsletter Contributors & Translations
+
+This is a community effort, and we would like to thank translators for their contributions. Volunteers are welcome! Translations of the XMPP Newsletter will be released here (with some delay):
+
+- English (original): [xmpp.org](https://xmpp.org/categories/newsletter/)
+  - General contributors: **Alexander "PapaTutuWawa", emus, Licaon_Kter, Ludovic Bocquet, MattJ, MSavoritias (fae,ve), wurstsalat, Zash** 
+- French: [jabberfr.org](https://news.jabberfr.org/category/newsletter/) and [linuxfr.org](https://linuxfr.org/tags/xmpp/public)
+  - Translators: **Adrien Bourmault (neox), alkino, anubis, Benoît Sibaud, Pierre Jarillon, Ppjet6, Ysabeau**
+- German: [xmpp.org](https://xmpp.org/categories/newsletter/) and [anoxinon.de](https://anoxinon.de/blog/)
+  - Translators: **Jeybe, wh0nix**
+- Italian: [nicfab.it](https://www.nicfab.it/)
+  - Translators: **nicfab**
+- Spanish: [xmpp.org](https://xmpp.org/categories/newsletter/)
+  - Translators: **daimonduff, TheCoffeMaker**
+
+## XSF Announcements
+
+If you are interested in becoming an XSF member, the [listings are open until this 19th February 2023](https://wiki.xmpp.org/web/Membership_Applications_Q1_2023)!
+
+### XSF fiscal hosting projects
+
+The XSF offers [fiscal hosting](https://xmpp.org/community/fiscalhost/) for XMPP projects. Please apply via [Open Collective](https://opencollective.com/xmpp). For more information, see the [announcement blog post](https://xmpp.org/2021/09/the-xsf-as-a-fiscal-host/). Current projects:
+
+- [Bifrost bridge: Offline messages exchange between Matrix and XMPP](https://opencollective.com/bifrost-mam)
+- [Mellium Co-op](https://opencollective.com/mellium)
+
+### Google Summer of Code 2022
+
+{{< figure src="/images/logos/GSoC_2022_Logo.png" caption="XSF and Google Summer of Code 2023" >}}
```

Comment:
image URL needs to be updated

Reply:
Can you help me to get the SVG in here correctly? :(

Reply:
Done!